kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgent.java

```diff
@@ -214,21 +349,36 @@ private void touch(File file) throws IOException {
      */
     public static void premain(String agentArgs) {
         int index = agentArgs.indexOf(':');
+        String[] args = agentArgs.split(":");
         if (index == -1) {
```

Comment:
nit: we could remove `index` and change this block to test the length of args to make a better exception than ArrayIndexOutOfBounds if we have too few args.